R/multi_class_diag_plots.R

```diff
@@ -71,32 +71,35 @@ plot_multiclass_pred_numcol <-
            alpha = 1,
            size = 1,
            prob_scaling = FALSE,
-           event_level = "first",
            prob_breaks = (2:9) / 10,
            prob_eps = 0.001) {
-    prob_name <- first_class_prob_name(dat, event_level, y_name)
-    # plotting
-    dat %>%
+    dat <- dat %>%
+      dplyr::select(-.pred_class) %>%
+      tidyr::pivot_longer(
+        cols = starts_with(".pred_"),
+        names_to = "predicted_class",
+        values_to = "predicted_probabilities"
+      )
+    dat <-  dat %>%
       dplyr::mutate(
-        !!prob_name :=
+        predicted_probabilities :=
```

Comment:
I don't believe you need to use `:=` anymore, now that you are using a regular named variable (rather than `!!`).